src/Explorer/Panes/SettingsPane/index.test.ts

```diff
@@ -0,0 +1,63 @@
+import { DatabaseAccount } from "../../../Contracts/DataModels";
+import { updateUserContext, userContext } from "../../../UserContext";
+
+describe("Settings Pane", () => {
+    describe("shouldShowQueryPageOptions()", () => {
```

Comment:
I like this test, but really its testing userContext.apiType. I would move it to userContext.test.ts

Reply:
done

Reply:
done